.htaccess

```diff
@@ -9,3 +9,5 @@ RewriteRule ^/?(\d+)/javadoc - [S=2]
 RewriteRule ^/?(\d+)/images/ - [S=1]
 RewriteCond $2 !=protocol
 RewriteRule ^/?(\d+)/([a-z]+)(\.html)? /$1/documentation#$2 [R=302,L,NE]
+RewriteCond %{REQUEST_FILENAME}.html -f
+RewriteRule ^(.*)$ %{REQUEST_FILENAME}.html
```

Comment:
Add a rewrite rule, it will serve `<REQUEST_FILENAME>.html` if the HTML file exists.

I do have a question, is this file being used in Production?

Reply:
I find it strange we have to edit this file and I wonder how it works on the website. I'll try to find out